Sources/DistributedCluster/Plugins/ClusterSystemSettings+Plugins.swift

```diff
@@ -21,6 +21,7 @@ public struct PluginsSettings {
     }
 
     internal var plugins: [BoxedPlugin] = []
+    internal var actorLifecycleHooks: [PluginActorLifecycleHook] = []
```

Comment:
the front of the name is good; but maybe `PluginActorLifecyclePlugin`?

Reply:
Still not sure, maybe more like `ActorLifecycleAwarePlugin`? 🤔

Reply:
or just `LifecyclePlugin`... 🤔

Reply:
I see usage of `ActorLifecycle` term in the system, so will just call `ActorLifecyclePlugin`.